test/join.bats

```diff
@@ -146,3 +146,20 @@ function is_podman_available() {
 	sudo podman rm -f $ID_1 $ID_2
 	sudo podman pod rm $POD_ID
 }
+
+@test "Test fill-mappings" {
+	run unshare -muinpfr --mount-proc true
+	if [[ "$status" -ne 0 ]]; then
+		skip "unshare doesn't support all the needed options"
+	fi
+
+	INTERVAL=10$RANDOM
+	unshare -muinpfr --mount-proc sleep $INTERVAL &
+
+        echo $INTERVAL
```

Comment:
Nit: line has different indentation.

Reply:
thanks for noticing it!  Pushed a fixed version